python/qpid_dispatch/management/qdrouter.json

```diff
@@ -1002,10 +1002,17 @@
                 "saslPassword": {
                     "type": "string",
                     "required": false,
-                    "description": "The password that the connector is using to connect to a peer.",
+                    "description": "(DEPRECATED) The password that the connector is using to connect to a peer.",
```

Comment:
Compare this to the text description for SSL's password.  SSL Password's description contains a bit more detail - can you update saslPassword's description to be consistent?

> This attribute has been deprecated because it is unsafe to store plain text passwords in config files. Use the passwordFile instead

Reply:
Done